recirq/dfl/dfl_2d_second_order_trotter.py

```diff
@@ -0,0 +1,751 @@
+import os
+import pickle
+from typing import cast, List, Sequence, Set, Tuple
+
+import cirq
+import numpy as np
+import numpy.typing as npt
+from tqdm import tqdm
+
+
+class LGTDFL:
```

Comment:
Names should avoid abbreviations:

https://google.github.io/styleguide/pyguide.html#316-naming